src/components/table/table.js

```diff
@@ -127,6 +131,38 @@ class Table extends PureComponent {
       .filter(c => firstColumnHeaders.includes(c))
       .concat(difference(activeColumnNames, firstColumnHeaders));
     const columnLabel = columnSlug => capitalize(columnSlug.replace(/_/g, ' '));
+
+    const getColumnLength = column => {
+      let samples = 0;
+      let aggregatedLenght = 0;
+      const sampleNumbersArray = Array
+        .from(Array(this.columnWidthSamples))
+        .map((a, index) => index);
+      sampleNumbersArray.forEach(n => {
+        if (data[n][column].length) {
+          aggregatedLenght += data[n][column].length;
+          samples += 1;
+        }
```

Comment:
are you a fan of curry functions :D ? I see a potential here for a curry function passing `sampleNumbersArray` and `data`